store/router.go

```diff
@@ -98,8 +91,8 @@ func (r *Router) Get(ctx context.Context, key []byte, cm commitments.CommitmentM
 		}
 
 		// 3 - read blob from fallbacks if enabled and data is non-retrievable from EigenDA
-		if r.fallbackEnabled() {
-			data, err = r.multiSourceRead(ctx, key, true)
+		if r.secondary.FallbackEnabled() {
+			data, err = r.secondary.MultiSourceRead(ctx, key, true, r.eigenda.Verify)
```

Comment:
Thinking through fallbacks some more. Do we really need them? I can't think of a use case when I would want a fallback instead of a cache. Why would I only want to read after eigenDA, instead of always read before eigenDA? Would greatly simplify the code if we could just get rid of fallbacks. But I might be missing something here still..